src/nvim/window.c

```diff
@@ -506,11 +506,23 @@ static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
  *
  * return FAIL for failure, OK otherwise
  */
-int win_split(int size, int flags)
+int win_split(int size, int flags) {
```

Comment:
7 with ```_tab_suc```, 21 without at the moment. But it seems I have forgotten ```:sball``` and ```:copen/:lopen```, so maybe 50/50 in the end.